core/templates/dev/head/domain/exploration/AnswerStatsObjectFactory.ts

```diff
@@ -17,9 +17,30 @@
  * particular answer from some particular state.
  */
 
-var oppia = require('AppInit.ts').module;
+import * as _ from 'lodash';
```

Comment:
Could we import just the part of `lodash` that we need as we discussed before? I'm okay with you creating an issue about this and fixing it later.

Reply:
`import * as cloneDeep from 'lodash/cloneDeep';` could probably work.

Reply:
Raised #7218 for this!